packages/wrangler/src/type-generation.ts

```diff
@@ -112,6 +112,47 @@ export async function typesHandler(
 	);
 }
 
+/**
+ * Check if a string is a valid TypeScript identifier. This is a naive check and doesn't cover all cases
+ * If not, it should be wrapped in quotes
+ */
+export function isValidIdentifier(key: string) {
+	return /^[a-zA-Z_$][\w$]*$/.test(key);
+}
+
+/**
+ * Construct a type key, if it's not a valid identifier, wrap it in quotes
+ */
+export function constructTypeKey(key: string) {
+	if (isValidIdentifier(key)) {
+		return `${key}`;
+	}
+	return `"${key}"`;
+}
+
+/**
+ *
+ * Construct a type key, if it's not a valid identifier, wrap it in quotes
+ * If rawVal is true, we'll use the value as the type, otherwise we'll format it for a string definition
+ */
+export function constructType(
+	key: string,
+	value: string | number | boolean,
+	rawVal = true
```

Comment:
`rawVal` seems like the function is receiving a raw value to me, I'd go with something like `useRawString`, `useVerbatinString`, etc... instead